executor/kvm_amd64.S

```diff
@@ -148,9 +148,9 @@ kvm_asm64_init_vm:
 	vmwrite %rax, %rdx;	\
 	/**/
 
-	VMSET_LIMITED(0x00004000, $0x3f, 0x481) // Pin-based VM-execution controls
-	//VMSET_LIMITED(0x00004002, $0x61999e84, 0x482) // Primary processor-based VM-execution controls
-	VMSET(0x00004002, $0xf3999e84)
+	VMSET_LIMITED(0x00004000, $0, 0x481) // Pin-based VM-execution controls
```

Comment:
`VMSET_LIMITED` does OR with the provided value, which is $0 in our case, so do we need this `VMSET_LIMITED` at all? or we can just delete it?
Or we still need it for the `and %rdx, %rax;` part? But I would assume it has some valid value if we don't touch it...

Reply:
Re-reading the Intel SDM, the new version looks fine. We are basically setting the maximum allowed value, right?

Reply:
Yes, we are setting the maximum allowed value. And we still need `VMSET_LIMITED` for the `and` part